### ISSUE TITLE
Orthographic Camera Array missing

### ISSUE THREAD

**Issue by Sythys** [mrdoob/three.js#15054]
ArrayCamera only extends and only supports Perspective Cameras.

**Comment by Mugen87** [comment]
It seems you can still use it with `THREE.OrthographicCamera`.

https://jsfiddle.net/f2Lommf5/15231/

**Comment by Sythys** [comment]
It is not missing but i did'nt got it runnig because the culling is off and i could not see my scene:
https://jsfiddle.net/f2Lommf5/15245/

**Comment by Sythys** [comment]
Did I configure somthing wrong? or is there a bug witch causes this?

**Comment by WestLangley** [comment]
@Sythys Sorry, this is not a help site. Please use the forum, instead.

We can leave this open as a feature request for `ArrayCamera` to support `OrthographicCamera`.

**Comment by mrdoob** [comment]
Yeah, currently `ArrayCamera` extends `PerspectiveCamera` and uses it for culling the objects being rendered by the sub-camera.

Out of curiosity... what's your use case?

**Comment by Sythys** [comment]
I want to loop a 2d Game scene on its edges. I dont want to duplicate the Objects. as any Object peaks the right boarder it should show up on the left. To create this effect i would have layered 9 cameras on top of each other to cover every edge and corner.

**Comment by mrdoob** [comment]
I see... For the time being, you should be able to increase the `ArrayCamera` `fov`.

```js
camera.fov = 100;
camera.updateProjectionMatrix();
```

https://jsfiddle.net/tgqznmsy/

**Comment by Sythys** [comment]
Thank you very much. This fixes my issue. :D

**Comment by harryhjsh** [comment]
I think this issue has been addressed by #30830 for `WebGPURenderer` thanks to the introduction of `FrustumArray` + the culling changes in `Renderer` from that PR. 

It obviously doesn't compute a new frustum, but does allow checking intersections for every camera in the array, which I think solves the spirit of what this issue describes?

Slightly tangentially: would a PR be acceptable to backport those culling changes using `FrustumArray` into `WebGLRenderer` as well? I think the structure in `render`/`_renderScene` and `projectObject` are similar enough that it should be trivial, so I think it's worthwhile (if not considered an overreaching enhancement).

**Comment by Mugen87** [comment]
Merging into https://github.com/mrdoob/three.js/issues/31672.
